### PR TITLE
API-diff between 7.0 RC2 and 7.0 GA

### PR DESCRIPTION
**Pull request by carlossanlop**
**Brace yourselves**: I will submit a _separate_ PR with the diff between 6.0 and 7.0.

If you see unusual diffs in this PR, please provide GitHub suggestions with the expected text. I would greatly appreciate it.

Currently excluding these attributes from showing up in the diff: [carlossanlop/arcade/AsmDiffUpdates/src/Microsoft.DotNet.AsmDiff/AttributesToExclude.txt](https://github.com/carlossanlop/arcade/blob/bc612740be9c302a65c3e57bdc2977b8af7c45d9/src/Microsoft.DotNet.AsmDiff/AttributesToExclude.txt). If you see any other attributes that you'd like me to exclude, let me know.

### ASP.NET

@dotnet/aspnet-api-review 

- System.Threading.RateLimiting: @rafikiassumani-msft @BrennanConroy @halter73 - Is it correct that the APIs showed up in the ASP.NET folder?

### WinForms

@merriemcgaw @RussKie - Nothing showed up, please let me know if that is correct.

### WPF

@singhashish-wpf and @pchaurasia14 - Nothing showed up, please let me know if that is correct.

### Libraries

@jeffhandley @karelz @ericstj @stephentoub

- System.Numerics: @dotnet/area-system-numerics 
- System.Text.Json: @dotnet/area-system-text-json 
- System.Transactions: @HongGit @roji

### CODE REVIEW

**Review by jeffhandley — APPROVED** [review]
`System.Numerics` and `System.Text.Json` changes match my expectations.

**Review by RussKie — APPROVED** [review]
To paraphrase @jeffhandley:

_Windows Desktop changes match my expectations._ :)

**Comment by mangod9** [comment]
@BrennanConroy for `System.Threading.RateLimiting`

**Comment by carlossanlop** [comment]
> @BrennanConroy for `System.Threading.RateLimiting`

My bad! I didn't see the [separate area owners](https://github.com/dotnet/runtime/blob/main/docs/area-owners.md) for this sub-namespace. I updated the description.

File: release-notes/7.0/preview/api-diff/ga/Microsoft.NETCore.App/7.0-ga_System.Numerics.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# System.Numerics
+
+``` diff
+ namespace System.Numerics {
```
Comment (tannergooding) [inline release-notes/7.0/preview/api-diff/ga/Microsoft.NETCore.App/7.0-ga_System.Numerics.md:4]: LGTM, just nullable annotations

CC. @dakersnar

**Review by eiriktsarpalis — APPROVED** [review]
System.Text.Json changes LGTM 👍

**Comment by pchaurasia14** [comment]
No changes for WPF. LGTM

**Review by roji — APPROVED** [review]
OK for System.Transactions

**Pull request merged**